support-frontend/app/controllers/PayPalRegular.scala

```diff
@@ -54,6 +54,16 @@ class PayPalRegular(
     }
   }
 
+  def createAgreementAndRetrieveUser: Action[Token] = maybeAuthenticatedAction().async(circe.json[Token]) { implicit request =>
```

Comment:
This is a new endpoint for the client side to hit, it retrieves the PayPal user details as well as the baid token which is needed by Zuora to bill the customer.